app/presenters/account_show_presenter.rb

```diff
@@ -36,7 +36,8 @@ def show_password_reset_partial?
   end
 
   def show_manage_personal_key_partial?
-    user.encrypted_recovery_code_digest.present? &&
+    (user.encrypted_recovery_code_digest_multi_region.present? ||
```

Comment:
I just want to make sure I understand this pattern everywhere: This probably should have been updated sooner, but it continued to work because everyone with a multi-region digest also had a single-region digest.

Do we have any 50/50 state concerns here? Like if a user suddenly doesn't have a single region digests on old hosts we suddenly assume they don't have a personal key?

Reply:
Yeah, exactly. And you're right, there would be a 50/50 state issue

Reply:
Opened https://github.com/18F/identity-idp/pull/11966 to address it and will have this PR branch off of that one to simplify the diff.